app/actors/sufia/create_with_files_actor.rb

```diff
@@ -46,10 +46,32 @@ def uploaded_files
       def mark_inactive
         return true unless Flipflop.enable_mediated_deposit?
         curation_concern.state = inactive_uri
+        send_inactive_message
       end
 
       def inactive_uri
         ::RDF::URI('http://fedora.info/definitions/1/0/access/ObjState#inactive')
       end
+
+      def send_inactive_message
+        return true if receiving_users == []
+        receiving_users.each do |admin|
+          message = "#{user.name} created <a href='#{work_path}'>#{curation_concern.title.first}</a>"
```

Comment:
Though the result returned will be different (joined titles rather than the first title), you could call `curation_concern.to_s` here instead of `curation_concern.title.first` and defer to the curation concern how it wishes to be coerced into a string.